src/commons/services/authService.js

```diff
@@ -28,7 +28,7 @@ export class AuthService {
   async loginAttempt(username, password) {
     try {
       const response = await this.httpClient.fetch(
-          `${this.baseUrl}auth/v1/login`,
+          `https://35.228.134.15:8443/api/auth/v1/login`,
```

Comment:
why are you hard-coding this? please revert this change. I assume this is some left over from your own deploy